tests/trainer/optimization/test_manual_optimization.py

```diff
@@ -28,54 +28,59 @@
 from tests.helpers.runif import RunIf
 
 
-@mock.patch.dict(os.environ, {"PL_DEV_DEBUG": "1"})
-def test_multiple_optimizers_manual_no_return(tmpdir):
-    """
-    Tests that only training_step can be used
-    """
+class AutomaticOptModel(BoringModel):
```

Comment:
why call it Automatic when it is doing the opposite? :)

Reply:
Derp